packages/e2e/alonzo-private-network/docker-compose.yml

```diff
@@ -0,0 +1,134 @@
+version: "3.3"
+
+services:
+  private-testnet:
+    build:
+      context: .
+    ports:
+      - 3001:3001
+    volumes:
+      - ${PWD}/sockets:/root/sockets
+      - ./config/network/${NETWORK:-mainnet}:/config/nodes # Mount the config so we can update the genesis files and hashes.
+    healthcheck:
+      # Verify that the node socket is up.
+      test: ["CMD-SHELL", "test - f /root/sockets/node-pool1.sock"]
+      interval: 10s
+      timeout: 5s
+      retries: 10
+  postgres:
+    image: postgres:${POSTGRES_VERSION:-11.5-alpine}
+    environment:
+      - POSTGRES_LOGGING=true
+      - POSTGRES_DB_FILE=/run/secrets/postgres_db
+      - POSTGRES_PASSWORD_FILE=/run/secrets/postgres_password
+      - POSTGRES_USER_FILE=/run/secrets/postgres_user
+    ports:
+      - ${POSTGRES_PORT:-5432}:5432
+    secrets:
+      - postgres_db
+      - postgres_password
+      - postgres_user
+    shm_size: '2gb'
+    restart: on-failure
+    healthcheck:
+      # Use pg_isready to check postgres is running. Substitute different
+      # user `postgres` if you've setup differently to config/pgpass-mainnet
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "200k"
+        max-file: "10"
+
+  cardano-node-ogmios:
+    image: cardanosolutions/cardano-node-ogmios:${CARDANO_NODE_OGMIOS_VERSION:-v5.1.0}-${NETWORK:-mainnet}
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "400k"
+        max-file: "20"
+    ports:
+      - ${OGMIOS_PORT:-1337}:1337
+    restart: on-failure
+    healthcheck:
```

Comment:
This is unnecessary, as the Ogmios Dockerfile [defines a health check ](https://github.com/CardanoSolutions/ogmios/blob/237b4ea9bc0312405af16afccda2277f9ae3f819/Dockerfile#L71)

Reply:
fixed here https://github.com/input-output-hk/cardano-js-sdk/pull/305/commits/31542f04deaa9785e179113e80e9e6a2205d1073